rust-runtime/aws-smithy-runtime/src/client/orchestrator.rs

```diff
@@ -156,30 +173,30 @@ async fn try_attempt(
     cfg: &mut ConfigBag,
     interceptors: &Interceptors,
 ) {
-    halt_on_err!([ctx] => interceptors.read_before_attempt(ctx, cfg).map_err(Into::into));
-    halt_on_err!([ctx] => orchestrate_endpoint(ctx, cfg).map_err(Into::into));
-    halt_on_err!([ctx] => interceptors.modify_before_signing(ctx, cfg).map_err(Into::into));
-    halt_on_err!([ctx] => interceptors.read_before_signing(ctx, cfg).map_err(Into::into));
+    halt_on_err!([ctx] => interceptors.read_before_attempt(ctx, cfg));
```

Comment:
It's just how the macro was written. They don't serve any purpose now. Previously when we had context _and_ `AttemptCheckpoint`, the presence of the `[]` determined which of those the macro was using. However, now that `AttemptCheckpoint` has been removed, they're just vestigial.

We are intentionally using `=>` instead of `,` though so that rustfmt doesn't bloat each of these lines to 4-5 lines of code.